Src/BootCamp.Chapter/Person.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
 {
     public class Person
     {
-        // add missing properties
+        public string Name { get; set; }
+        public string BirthDay { get; set; }
+        public string SureName { get; set; }
+        public string Gender { get; set; }
+        public string Country { get; set; }
+        public string Email { get; set; }
+        public string StreetAdress { get; set; }
+
+        public int CalculateAge()
+        {
+            CultureInfo culture = new CultureInfo("en-US");
+            var convertedBirthday = DateTime.Parse(this.BirthDay, culture).Year;
```

Comment:
What if someone was born 5th May 1990 and the current date is 2nd Feb 2020. What is their age?

Really they are 29 as it's not been past 5th May. but your code will produce 30 as `2020-1990 = 30`

Probably not so much of an issue for this homework - but just food for thought :)

Reply:
oke, I have then to find a better way to calcuate the age. I hope google can help me with that.

Reply:
found it